tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -47,6 +46,8 @@
 from robottelo.rhsso_utils import update_rhsso_user
 from robottelo.utils.issue_handlers import is_open
 
+# from robottelo.decorators import skip_if_not_set
```

Comment:
seems like this is not required

Reply:
I could remove that but I as it was used in the file and was essential, I thought of keeping it commented.

Reply:
No value to leaving commented imported statements, please remove.  These are providing no context as to how/where/why they _might_ be needed.